Интеграция форм с midi

fostfox opened the pull request:
Подключение всех форм к основному окну приложения.

fostfox committed:
Подключеник всех форм к главному окну

fostfox committed:
Мелкие правки модуля БД

fostfox committed:
Уборка мусора

fostfox committed:
Реализация отсутствующих функций

ArnMips commented:
## Найдены проблемы
- [ ] Изменить названия всех форм с Form, на подходящее название
- [ ] Отредактировать валидаторы проверок при редактировании (сейчас позволяют вводить только **один** символ, и не позволяют вводить **дробные** значения)
- [ ] Исправить слово "По углеродам" на "По углеводам" в формах поиска
- [ ] Исправить в форме поиска по Активностям для виджета `stackedWidget` индекс по умолчанию (иначе там постоянно диапазон высвечивается) 
- [ ] Проверить форму информации о Рецепте (сейчас там отображается не вся информация). Кроме того отключить воз-ть редактирования информации в таблице. **Возможно таблицу воjбще заменить на массив QLable?**
- [ ] Проверить форму редактирования Рецепта. Там явно не хватает необходимой логики для работы всей функциональности (как минимум поиска продуктов для добавления ). Вообще очень хорошо пересмотреть дизайн этой формы, так как не все логично, и понятно, и адекватно тут сделано
- [ ] Отредактировать диалоговое окно _О программе_. (добавить как минимум номер версии программы)

ArnMips commented:
## Найдены проблемы
- [ ] Изменить названия всех форм с Form, на подходящее название
- [ ] Отредактировать валидаторы проверок при редактировании (сейчас позволяют вводить только **один** символ, и не позволяют вводить **дробные** значения)
- [ ] Исправить слово "По углеродам" на "По углеводам" в формах поиска
- [ ] Исправить в форме поиска по Активностям для виджета `stackedWidget` индекс по умолчанию (иначе там постоянно диапазон высвечивается) 
- [ ] Проверить форму информации о Рецепте (сейчас там отображается не вся информация). Кроме того отключить воз-ть редактирования информации в таблице. **Возможно таблицу воjбще заменить на массив QLable?**
- [ ] Проверить форму редактирования Рецепта. Там явно не хватает необходимой логики для работы всей функциональности (как минимум поиска продуктов для добавления ). Вообще очень хорошо пересмотреть дизайн этой формы, так как не все логично, и понятно, и адекватно тут сделано
- [ ] Отредактировать диалоговое окно _О программе_. (добавить как минимум номер версии программы)

Galoev committed:
Были исправлены названия форм
Убраны валидаторы, чтобы можно было написать любое название
Исправлено слов "По углеродам" на "По углеводам" в формах поиска
Исправлена начальная форма поска в форме поиска Активности, вместо поиска по диапазону, по умолчанию теперь поиск по названию
Добавлена логика незватающия для формы редактирования Рецептов, также форма рецептов дублирует/перенаправляет сигнлы от формы поиска продуктов

fostfox committed:
Фикс добавления Активности в БД [#10]

Galoev committed:
Исправлена фомрма информации о Рецепте
Отключена возможность редактирования таблицы
Столбцы таблицы теперь увеличиваются вместе с основным окном

Galoev committed:
Запрещена возможность редактирования таблицы продуктов в форме редактирования рецептов

Galoev committed:
Исправлена проблема с закрытием форм по нажатию на кнопку отмена

fostfox committed:
Возможность обновления объектов в БД [#4]

 - Поддержка обновления информации об объектах типа `ProductEntity` и  `ActivityEntity`

fostfox committed:
Merge branch 'iss4-2' into gui-midi

fostfox committed:
v0.1.1

fostfox committed:
Возможность обновления объектов в БД v2 [#4]

 - Исправление багов при попытках изменения информации для объектов типа `ProductEntity` и  `ActivityEntity`
 - Поддержка обновления информации для объектов типа `RecipeEntity`

fostfox committed:
Merge branch 'iss4-2' into gui-midi

fostfox committed:
Merge branch origin 'gui-midi' of into gui-midi

fostfox committed:
Очистка

fostfox committed:
Авто создание файла БД [#9]

 - Автоматическое создание директории и файла БД, если они отсутствуют
 - Инициализация созданного файла для БД постыми таблицами

fostfox committed:
Обновление скрипта сборки

fostfox committed:
v0.1.2

Galoev committed:
Nothing changed

Galoev committed:
Merge branch 'gui-midi' of https://github.com/Ilkin-Galoev/nutritionist-helper into gui-midi

Galoev committed:
Исрпавлены валидаторы

fostfox committed:
Поиск по бжук [#7] и мелкие исправления

 - Добавление части логики в MIDI для работы с формой RecipeSearch

Galoev committed:
Исправлены мелкие ошибки 
Добавлена логика для формы редактирования Рецепта (установка продуктов , метод setProducts)  
Исправлена ошибка с отображение дробных значений в информации о продукте  
Исправлена проблема с полным закрыванием суб-окон

Galoev committed:
Merge branch 'gui-midi' of https://github.com/Ilkin-Galoev/nutritionist-helper into gui-midi

fostfox committed:
Добавлен фон в Mdi

 - теперь клиенту понятно, что программа успешно запущена и это не просто
 серый фон загрузки программы

fostfox committed:
Фикс форм рецептов

 - Исправление поиска продуктов
 - Корректный расчет бжук для рецептов

Galoev committed:
В форме редактирования продукта добавлена возможно добавления в чем измеряется этот продукт и в форме информации эта информация отображается
